zer0-os/zOS
# feat: conversation header alignment and style

### What does this do?
- Alters the conversations headers to use the same text, icon size and positioning to ensure a smooth transition between the stages of a flow.
- Hides the Rewards Bar when starting one of the flows i.e. starting a group converstation.
- Updates the tests.

### Why are we making this change?
- Right now, when creating a new conversation in the app, the position of the header shifts. See the attached gif for an illustration of the difference in positioning between “Conversations” and “New Message.”

### How do I test this?
- open messenger and in the sidekick panel check the panel header when navigating through stages

### Key decisions and Risk Assessment:
  #### Things to consider:
  1. How will this affect security?
  1. How will this affect performance?
  1. Does this change any APIs?



https://github.com/zer0-os/zOS/assets/39112648/36f02cb0-4531-46bf-b5c3-2da333ba584a
